src/features/profile/components/Profile.tsx

```diff
@@ -173,13 +181,13 @@ export const Profile: FC<ProfileProps> = ({user}) => {
         </div>
       </div>
 
-      <InlineMenu />
+      <InlineMenu action={notImplemented} />
 
       <hr className="mx-6" />
 
       <div className="grid h-auto grid-cols-8 gap-4 p-10">
         {posts.map((post) => (
-          <CustomCard key={post.id} user={user} post={post} />
+          <CustomCard key={post.id} post={post} isSelf={isSelf} />
```

Comment:
move Custom card in features/post/components/UserPostCard.tsx

you don't need to pass isSelf prop, just do the same thing we did to get 'isSelf' value in the component